explainaboard/analysis/analyses.py

```diff
@@ -113,6 +113,16 @@ def from_dict(dikt: dict):
                 features=tuple(dikt['features']),
             )
 
+    @staticmethod
+    def _subsample_analysis_cases(
+        sample_limit: int, analysis_cases: list[int]
+    ) -> list[int]:
```

Comment:
Add docstring of this function so that other people can understand what this is.